### ISSUE TITLE
Support multi-shot 

### ISSUE THREAD

**Issue by MushiTheMoshi** [flameshot-org/flameshot#1137]
Hi there, awesome tool. I'm using it daily. Great work. I was looking into gnome-screenshot and scrot and both support multi-screen shot:
so my suggestion are:
- ability to take multiple images in a period of time 
- ability to setup the time and how many shots 
- ability to gather those images and merge into one large

I know you are probably working in some other awesome features, though I believe these are really needed. 
Regards,
Julio

**Comment by borgmanJeremy** [comment]
Do you mean a feature for like creating gifs?

**Comment by MushiTheMoshi** [comment]
No, mutiple images (png/jpeg) in one click, a gif it'd be an animation.

ability to take multiple images in a period of time <<< useful to gather  images in movement 
ability to setup the time and how many shots       <<< same as before but for larger or shorter
ability to gather those images and merge into one large  <<< useful to gather multiple images like pdf, html in one single image

Regards!
